### ISSUE TITLE
Status effects (Including monster tp moves) ignore players elemental resistance special mechanic

### ISSUE THREAD

**Issue by ShiyoKozuki** [DerpyProjectGroup/topaz#826]
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened
- [x] I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x] I have read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I have specified what branch this happens on branch: EDITME

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 
tldr: If you have ANY negative resist, a status effect(of that negative element) will ALWAYS, no matter if it's a level1 rabbit and you're level 99 with godmode on 20 times, land on you for at least half duration.
You cannot 1/8th resist without having at least 1 resist(to that element), again, no matter mob levels, your levels, your stats, etc.
Too much to type, read up here and next few pages.
https://www.ffxiah.com/forum/topic/46016/first-and-final-line-of-defense-v20/107/#3571049
https://www.ffxiah.com/forum/topic/46016/first-and-final-line-of-defense-v20/107/#3570888

**Comment by TeoTwawki** [comment]
I think you wanted to link https://www.ffxiah.com/forum/topic/46016/first-and-final-line-of-defense-v20/107/#3571049 which actually talks about how his magic eva was doing dick till he put on a small amount of elemental resist
